### PR TITLE
3.7 Added local ports column to syscollector table

### PR DESCRIPTION
**Pull request by manuasir**
Hi, this PR adds a column with the `local ports` field from the `syscollector` API endpoint.

Related issue: #354 

Regards

### CODE REVIEW

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/inventory/inventory.html [hunk 1]
```diff
@@ -93,7 +93,7 @@
       <span class="wz-headline-title"><i class="fa fa-fw fa-exchange"></i> Network ports</span>
       <span class="color-grey pull-right">Last scan: {{syscollector.ports.items[0].scan.time}}</span>
       <md-divider class="wz-margin-top-10"></md-divider>
-      <wazuh-table flex path="'/syscollector/' + agent.id + '/ports'" row-sizes="[4]" keys="[{value:'local.ip',nosortable:true},{value:'remote.ip',nosortable:true},'state','protocol']">
+      <wazuh-table flex path="'/syscollector/' + agent.id + '/ports'" row-sizes="[4]" keys="[{value:'local.ip',nosortable:true},{value:'local.port',nosortable:true},{value:'remote.ip',nosortable:true},'state','protocol']">
```
Comment (jesusgn90) [inline SplunkAppForWazuh/appserver/static/js/controllers/agents/inventory/inventory.html]: `remote.port` is missing.

Also check if those fields are sortable please.

File: SplunkAppForWazuh/appserver/static/js/services/key-equivalence/keyEquivalenceService.js [hunk 2]
```diff
@@ -56,8 +56,9 @@ define(['../module'], function(module) {
           proto: 'Protocol',
           address: 'Address',
           protocol: 'Protocol',
-          'local.ip': 'Local',
-          'remote.ip': 'Remote'
+          'local.ip': 'Local IP',
+          'remote.ip': 'Remote',
```
Comment (jesusgn90) [inline SplunkAppForWazuh/appserver/static/js/services/key-equivalence/keyEquivalenceService.js]: Remote IP

File: SplunkAppForWazuh/appserver/static/js/services/key-equivalence/keyEquivalenceService.js [hunk 3]
```diff
@@ -56,8 +56,9 @@ define(['../module'], function(module) {
...
+          'local.port': 'Local port'
```
Comment (jesusgn90) [inline SplunkAppForWazuh/appserver/static/js/services/key-equivalence/keyEquivalenceService.js]: Add `remote.port` please

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/inventory/inventory.html (hunk 1 above)
Comment (manuasir) [reply SplunkAppForWazuh/appserver/static/js/controllers/agents/inventory/inventory.html]: They're not currently sortable.

**Pull request merged**